### PR TITLE
move csv pass through path [resolves #36]

### LINKED ISSUE #36: When a path to a csv is given, only read first line
Only the first line of a csv is needed to check metta's assumptions about the file. Reading the whole dataframe with pd.read_csv is inefficient. Would be preferable to simply load the first line of the csv and then copy or move the file to the new destination.

### PR DESCRIPTION
**Pull request by not-the-fish**
Fixes #36 

Changes made in this Pull Request:
When a csv path is sent to `archive_matrix`:
 - do not read into a pandas dataframe
 - read the first line and check that last element is label
 - copy the csv to the new location

 PR Checklist
 ------------
  - [ ] Tests?
  - [ ] Docs?
  - [x] Issue raised/referenced?

### CODE REVIEW

File: metta/metta_io.py [hunk 1]
```diff
@@ -117,14 +119,14 @@ def archive_matrix(
             raise IOError('Not a file: {}'.format(abs_path_file))
 
         if abs_path_file[-3:] == 'csv':
-            df_matrix = pd.read_csv(abs_path_file)
```
Comment (thcrock) [inline metta/metta_io.py:118]: Idea: to keep most of this code similar, is it possible to make a one-line dataframe here instead? I recalled that pandas.read_csv has a chunk size, and maybe it also has a row limit...

Just checked the docs. It looks like there is an nrows argument to read_csv, which I think we could use here.

Related: although the pandas read_hdf function has a far different interface, it looks it has something similar: start and stop. So maybe we could use that too?

**Comment by not-the-fish** [comment]
I went with the csv class because my impression is pd.read_csv is
relatively slow, and I'd like to squeeze as much speed improvement as I can
from this part of the pipeline. But it's worth benchmarking to see how much
difference it makes.

**Comment by avishekrk** [comment]
pd.read_csv is the fastest at reading csv's in the python world https://softwarerecs.stackexchange.com/questions/7463/fastest-python-library-to-read-a-csv-file

**Comment by not-the-fish** [comment]
alrighty! i support the change. :) i can fix it tonight.

**Comment by thcrock** [comment]
Speed, for sure, but also worth thinking about is memory. I did try read_csv with nrows=1 on a 2GB CSV earlier today, and not only did it happen instantaneously but it didn't use much memory either.

**Comment by not-the-fish** [comment]
nice! i was overestimating how alert i'd be tonight. will push an update
tomorrow morning :)

**Comment by not-the-fish** [comment]
Ok, i think that fixes it

**Pull request merged**